packages/typespec-python/test/azure/generated/typetest-property-optional/typetest/property/optional/operations/_operations.py

```diff
@@ -5107,8 +5100,7 @@ def get_default(self, **kwargs: Any) -> _models.UnionIntLiteralProperty:
 
                 # response body for status code(s): 200
                 response == {
-                    "property": 1  # Optional. Default value is 1. Property. Is either a
-                      Literal[1] type or a Literal[2] type.
+                    "property": 0  # Optional. Property. Known values are: 1 and 2.
```

Comment:
The default value is 0 while known values are "1" and "2"...

Reply:
Is it the problem of example generation of enum? I haven't changed related code.